component/wallet-cli/cmd/oidc4ci.go

```diff
@@ -166,14 +184,23 @@ func NewOIDC4CICommand() *cobra.Command {
 	cmd.Flags().StringVar(&flags.CredentialFormat, "credential-format", "", "credential format")
 	cmd.Flags().StringVar(&flags.Pin, "pin", "", "pre-authorized flow pin")
 	cmd.Flags().BoolVar(&flags.Debug, "debug", false, "enable debug mode")
+	cmd.Flags().BoolVar(&flags.InsecureTls, "insecure-tls", false, "disable certificate validation for http client")
```

Comment:
can you plz name it "--skip-tls" or just "--insecure" like in curl, "insecure-tls" sounds strange 😕

Reply:
@aholovko done